.github/workflows/SubmitReview.yml

```diff
@@ -0,0 +1,113 @@
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const TODO_COLUMN = 4971951;
+            const IN_PROGRESS_COLUMN = 4971952;
+            const REVIEW_IN_PROGRESS_COLUMN = 4971953;
+            const REVIEW_APPROVED_COLUMN = 4971954;
+            const DONE_COLUMN = 4971955;
+            //
+            async function getIssue(issue_number) {
+                try {
+                    return (await github.issues.get({
+                        owner: context.repo.owner,
+                        repo: context.repo.repo,
+                        issue_number
+                    })).data;
+                }
+                catch (error) {
+                    console.log(`Issue #${issue_number} not found: ${error}`);
+                    return null;
+                }
+            }
+            //
+            async function findCard(content_url) {
+                // Columns are searched from the most probable one
+                const allColumns = [REVIEW_IN_PROGRESS_COLUMN, REVIEW_APPROVED_COLUMN, IN_PROGRESS_COLUMN, TODO_COLUMN, DONE_COLUMN];
+                for (let i = 0; i < allColumns.length; i++) {
+                    let cards = await github.projects.listCards({ column_id: allColumns[i] });
+                    let card = cards.data.find(x => x.content_url == content_url);
+                    if (card) {
+                        return card;
+                    }
+                }
+                console.log("Card not found for: " + content_url);
+                return null;
+            }
+            //
+            async function removeAssignees(issue){
+                const oldAssignees = issue.assignees.map(x => x.login);
+                if (oldAssignees.length !== 0) {
+                    console.log("Removing assignees: " + oldAssignees.join(", "));
+                    await github.issues.removeAssignees({
+                        owner: context.repo.owner,
+                        repo: context.repo.repo,
+                        issue_number: issue.number,
+                        assignees: oldAssignees
+                    });
+                }
+            }
+            //
+            async function addAssignee(issue, login) {
+                console.log("Assigning to: " + login);
+                await github.issues.addAssignees({
+                    owner: context.repo.owner,
+                    repo: context.repo.repo,
+                    issue_number: issue.number,
+                    assignees: [login]
+                });
+            }
+            //
+            async function processIssue(issue) {
+                if (issue.state == "open") {
+                    removeAssignees(issue);
+                    addAssignee(issue, context.payload.pull_request.user.login);
+                    const card = await findCard(issue.url);
+                    const newColumn = context.payload.review.state == "approved" ? REVIEW_APPROVED_COLUMN : IN_PROGRESS_COLUMN;
```

Comment:
this function should also check the number of reviewers - if there are 2 reviewers, one approved and one requested changes, the card should not be moved to Reviewer Approved

Reply:
I gave it couple of hours but the logic gets too complicated 😞  Let's keep it like this as a baby step and see if we need to fix it or not.

Reply:
I added a note about this idea to my Kanban backlog.